lib/Providers/Qr/BaconQrCodeProvider.php

```diff
@@ -52,22 +51,17 @@ public function getMimeType(): string
         throw new RuntimeException(sprintf('Unknown MIME-type: %s', $this->format));
     }
 
-    public function getQRCodeImage(string $qrText, int $size): string
+    public function getQRCodeImage(string $qrtext, int $size): string
```

Comment:
camelCase should be used here

Reply:
aha I'd vote for camelCase everywhere in that case!

Reply:
Here you go, fixed in https://github.com/RobThree/TwoFactorAuth/pull/108/commits/e554a9b5e482d9e8e9b72353d8e23501e663607c :)